docs/reference/datamodel/links.rst

```diff
@@ -855,9 +855,7 @@ The valid SDL sub-declarations are listed below:
     The expression must be :ref:`Stable <ref_reference_volatility>`.
 
 :eql:synopsis:`readonly := {true | false}`
-    If ``true``, the link is considered *read-only*.  Modifications
-    of this link are prohibited once an object is created.  All of the
-    derived links **must** preserve the original *read-only* value.
+    If ``true``, the link is considered *read-only*.  Modifications of this link using ``update`` are prohibited once an object is created. All of the derived links **must** preserve the original *read-only* value. Changes to this link **can** occur if a link is deleted and the appropriate deletion policy allows it.
```

Comment:
```suggestion
    If ``true``, the link is considered *read-only*.  Modifications of this link using ``update`` are prohibited once an object is created. All of the derived links **must** preserve the original *read-only* value. Changes to this link **might** still occur if the target object is deleted and the deletion policy allows it.
```

Reply:
@aljazerzen I agree with everything except the change from "can" to "might still", since if you _have_ deleted a target object with the appropriate deletion policy, it _will_ update the link set. wdyt?

Reply:
I guess the language we're using here isn't completely clear: maybe "will" is better than "can" or "might still"?

Reply:
Why do we have to use vague "can" / "might" instead of a concrete "will"?

Reply:
I wanted to say that 'even on "required", it is not guaranteed that the value will not change'. So the value "might still change".

But "will" also conveys the same thing.